ports/yolkheroes/README.md

```diff
@@ -0,0 +1,20 @@
+## Notes
+
+Please support the developers at https://store.steampowered.com/app/2417850/?snr=1_5_9__205 or https://pixelwestern.itch.io/yolk-hero. Purchasing the game is mandatory to run it.
+
+Yolk Heroes is a Tamagotchi-style idle game where the player raises an Elf to become a hero. Think Digimon Tamagotchi meets Princess Maker as an idle game.
+
+The Gameboy-style art and screen scaling make it a good candidate for a portable platform.
+
+## Controls
+
+| Button | Action |
+|--|--| 
+|A|'Confirm'|
+|B|'Back'|
+|X|'Confirm'|
+|Y|'Back'|
+|Start|'Menu'|
+|Select|'Space'|
+
+Please see https://github.com/PortsMaster/gptokeyb for a guide on text input.
```

Comment:
It's simple enough to just add a few lines to tell the user how to use text input. They're far less likely to get past the name screen if they have to go to another page to find out what to do.